scripts/dnscheck.py

```diff
@@ -0,0 +1,22 @@
+import dns.resolver
+
+def check_domain(domain):
+    try:
+        # Query for 'A' records
+        answers = dns.resolver.resolve(domain, 'A')
+        # If we get any answers, the domain has a response
+        return bool(answers)
+    except (dns.resolver.NoAnswer, dns.resolver.NXDOMAIN, dns.resolver.Timeout):
```

Comment:
Only NXDOMAIN is enough. Definitely not timeout as it is a DNS server issue. No Answer has a lot of weird cases so I don't know when it should be honored.

We should prevent losing active domains in this checkup. Thus again, I think NXDOMAIN is enough.

Reply:
Thanks, done!